### ISSUE TITLE
learning rate scheduler

### ISSUE THREAD

**Issue by raminrahimi6970** [clovaai/deep-text-recognition-benchmark#347]
hi thanks for your code and best paper
I do not see learning rate scheduler in training loop,it don not affect in performance of model?

**Comment by nguyenviettuan96** [comment]
learning rate was set to 1 and it don't change during training process

**Comment by nguyenviettuan96** [comment]
but it affect to optimizer
